src/common/TALENTS/evoker.ts

```diff
@@ -1324,6 +1324,14 @@ const talents = {
     entryIds: [115564],
     definitionIds: [{ id: 120576, specId: 1468 }],
   },
+  TIME_OF_NEED_SUMMON: {
```

Comment:
Keep this in SPELLS, this file gets re-generated on patches so it will get removed automatically.